splice_machine/src/main/java/com/splicemachine/derby/stream/ActivationHolder.java

```diff
@@ -165,6 +168,7 @@ public synchronized void writeExternal(ObjectOutput out) throws IOException {
             out.writeObject(groupUsers);
         } else
             out.writeBoolean(false);
+        out.writeObject(getActivation().getLanguageConnectionContext().getDataDictionary().getDataDictionaryCache().getPropertyCache());
```

Comment:
This enhancement is targetted for Spark, not control, right?  Maybe we should only serialize the property cache if running on Spark so control doesn't take a performance hit.

Reply:
@msirek Yes it is for Spark. But isn't the serialization for ActivationHolder is only for Spark?

Reply:
I saw ActivationHolder in ControlOperationContext... but I tried testing control queries and don't see SerDe happening, so I guess that part is spark only.